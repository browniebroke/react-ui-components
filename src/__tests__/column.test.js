import React from 'react'
import renderer from 'react-test-renderer'

import Column from '../column'
import styledComponentsTheme from '../styled-components-theme'
import { ThemeProvider } from 'styled-components'

describe('Column component', () => {
  test('should render correctly without props', () => {
    let tree = renderer
      .create(
        <ThemeProvider theme={styledComponentsTheme}>
          <Column />
        </ThemeProvider>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })

  test('should render correctly with some props', () => {
    let tree = renderer
      .create(
        <ThemeProvider theme={styledComponentsTheme}>
          <Column xsWidth="50%" smWidth="33%" mdWidth="25%" lgWidth="20%" />
        </ThemeProvider>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
