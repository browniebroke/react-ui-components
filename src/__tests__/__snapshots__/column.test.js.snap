// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Column component should render correctly with some props 1`] = `
<div
  className="sc-bdfBwQ iWBFiY"
/>
`;

exports[`Column component should render correctly without props 1`] = `
<div
  className="sc-bdfBwQ iWBFiY"
/>
`;
